README.md

```diff
@@ -15,6 +15,20 @@ git clone https://github.com/mvelazc0/PurpleSpray.git
 pip install -r PurpleSpray/requirements.txt
 ```
 
+ ### Docker Build
+
+ ```
+ $ docker build -t xshuden/purplespray .
+ ```
+
+### Docker Usage
+
+ ```
+ $ docker run --rm -it xshuden/purplespray
```

Comment:
Thanks for the PR !!

Does this command need the "xshuden"  ?

Reply:
No not. It can be build with any name.

Reply:
Ex Build: docker build -t purplespray .
Ex Run: docker run --rm -it purplespray

Reply:
Thanks. I will change that part, test docker and merge the PR.

Reply:
Thanks.  You can test it here. https://hub.docker.com/r/xshuden/purplespray